data/human/free worlds 4 epilogue.txt

```diff
@@ -144,6 +144,52 @@ mission "FW Epilogue: Freya"
 				`	"You sound frustrated."`
 				`	"You know, the war is over, you can relax if you want to."`
 			`	"I know, I know," she says. "I complain, but really there's nowhere else I would rather be. Even if everything we're discovering here is nothing more than what they intended for us to discover, it's still the most interesting work I could possibly be doing right now. So, don't feel sorry for me. Really."`
+			branch surname
+				has "last name: Winters"
+			`	You talk for a while longer, and she tells you she'll be sure to be in touch if they ever get to the root of the Pug's motivation. "Until then," she says, "good luck in your travels. I hope the galaxy never runs out of ways to surprise you."`
+				decline
+
+			label surname
+			`	"By the way, can I ask you something, <first>? About you? Sorry if this rather abrupt. I've been too busy to talk during the war."`
+			choice
+				`	"Sure. What is it?"`
+					goto permitted
+				`	"Sorry, I'd prefer if you didn't."`
+					goto conclusion
+			
+			label permitted
+			`	"Do you happen to know anyone named Lavinia? Married to someone named Marcus?"`
+			`	Both names sound oddly familiar, but you can't remember where you heard them.`
+			choice
+				`	"I think so, but I need a minute to remember exactly who."`
+					goto heritage
+				`	"No, I don't."`
+					goto denial
```

Comment:
```suggestion
```

Reply:
Committed it, but wouldn't deleting this prevent the game from going to "Oh, okay. I think I've been mistaken, then. Never mind." when selecting "no"?

Reply:
No, because the label is right underneath it.
But it probably doesn't matter.